Dockerfile

```diff
@@ -1,4 +1,11 @@
-FROM openjdk:8u121-jdk
+FROM node:8
```

Comment:
Now using node as the base image, with openjdk installed on top of it. Actually this is an upgrade as well as jessie has been replaced by stretch.